app/serializers/v1/workflow/workflow_serializer.rb

```diff
@@ -1,10 +1,13 @@
 class WorkflowSerializer
   include JSONAPI::Serializer
 
-  set_type :movie  # optional
-  set_id :owner_id # optional
-  attributes :name, :year
-  has_many :processes
-  belongs_to :owner, record_type: :user
-  belongs_to :movie_type
+  attributes :name, :description
+
+  attribute :processes do |object|
+    object.processes.map do |process|
+      { title: process.title, effort: process.effort, status: process.status, categories: process.categories }
+    end
+  end
+
+  link :url
```

Comment:
where does the value of URL come from?

Reply:
i just made some dummy url function as a placeholder, until i figure out routes